public/app/actions/actions.js

```diff
@@ -4,10 +4,10 @@ var actions = Reflux.createActions([
   // user actions
   'addDevice',
   'removeDevice',
-  //room actions
+  // room actions
   'addRoom',
   'removeRoom',
-  //settings actions
+  // settings actions
```

Comment:
+1 for adding a space between `//` and the comment text. Would love if everybody does it as it's more readible